mne/cov.py

```diff
@@ -386,66 +389,90 @@ def compute_raw_covariance(raw, tmin=None, tmax=None, tstep=0.2,
     picks : array-like of int | None (default None)
         Indices of channels to include (if None, all channels
         except bad channels are used).
+    keep_sample_mean : bool (default False)
```

Comment:
Yes, exactly. I think somehow this option should even be deactivated in
continous mode ...

On Mon, Feb 15, 2016 at 4:47 PM, Eric Larson <notifications@github.com>
wrote:

> In mne/cov.py
> <https://github.com/mne-tools/mne-python/pull/2868#discussion_r52913551>:
>
> > @@ -386,66 +389,90 @@ def compute_raw_covariance(raw, tmin=None, tmax=None, tstep=0.2,
> >      picks : array-like of int | None (default None)
> >          Indices of channels to include (if None, all channels
> >          except bad channels are used).
> > +    keep_sample_mean : bool (default False)
>
> So instead it should probably read "the average response over epochs is
> computed and subtracted during the covariance computation"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2868/files#r52913551>.
>